perllib/FixMyStreet/Cobrand/Buckinghamshire.pm

```diff
@@ -194,6 +194,8 @@ sub open311_pre_send {
 sub open311_post_send {
     my ($self, $row, $h) = @_;
 
+    $self->_add_claim_auto_response($row, $h) if $row->category eq 'Claim';
```

Comment:
I /think/ this can be removed entirely, if the Claim template is made auto-response, then the default handling should work? This must be a hangover from before that changed to create-on-creation but confirm-on-submission, I think. But comparing the code here with what create_related_things does, it appears to be identical apart from requiring the Claim template to have 'auto response' ticked.

Reply:
@dracos I've had a dig around and it seems that the tests are expecting the claims response template to be included in the initial email, rather than be sent out separately as an update:

https://github.com/mysociety/fixmystreet/blob/0d8725ce773d08a4145b9f0dc04cfa2e0e3002f0/t/app/controller/claims.t#L221-L224

(The "is a lengthy process" bit comes from the claims response template in the tests)

The `_add_claim_auto_response` code is adding the update to `$h->{update}` which is then picked up by `templates/email/buckinghamshire/other-reported-claim.{html,txt}` to be included in the confirm_report_sent email, and then marking the alert as sent so it's not sent out again.

https://github.com/mysociety/fixmystreet/blob/0d8725ce773d08a4145b9f0dc04cfa2e0e3002f0/perllib/FixMyStreet/Cobrand/Buckinghamshire.pm#L250-L268

See https://github.com/mysociety/fixmystreet/commit/7aa154e565cc6e4eafc5884d5c2b6d4f6e781dc5 for the commit which introduced this change.

So I _think_ this code is still needed, in order to include the update in the initial email and to skip the subsequent alert.

Reply:
(I wonder if the code could be combined a bit then, but no need at this point)